grafana-plugin/src/pages/users/Users.tsx

```diff
@@ -213,7 +220,7 @@ class Users extends React.Component<UsersProps, UsersState> {
                     </div>
 
                     <GTable
-                      emptyText={results ? 'No users found' : 'Loading...'}
+                      emptyText={initialUsersLoaded ? 'No users found' : 'Loading...'}
```

Comment:
if no REQUIRED_PERMISSION_TO_VIEW_USERS then let's display some appropriate message like 
`generateMissingPermissionMessage(REQUIRED_PERMISSION_TO_VIEW_USERS)`
instead of `'No users found'`

Reply:
`GTable` won't get rendered if `!authorizedToViewUsers`, see [here](https://github.com/grafana/oncall/pull/1185/files#diff-9b30c8007fa630a2cafb31b17346113da5c47ded8b031c3e283e65c8cf7de99eR204) and the screenshots in the PR description